aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -99,181 +106,128 @@ The preceding command adds the [aspnet-codegenerator scaffolding tool](xref:fund
-
-```csharp
-using MvcMovie.Data;
-using Microsoft.EntityFrameworkCore;
-```
-
-Add the following highlighted code in `Startup.ConfigureServices`:
-
-# [Visual Studio](#tab/visual-studio)
-
-[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Startup.cs?name=snippet_ConfigureServices&highlight=5-6)]
-
-# [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
-
-[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Startup.cs?name=snippet_UseSqlite&highlight=5-6)]
-
----
-
-The name of the connection string is passed in to the context by calling a method on a [DbContextOptions](/dotnet/api/microsoft.entityframeworkcore.dbcontextoptions) object. For local development, the [ASP.NET Core configuration system](xref:fundamentals/configuration/index) reads the connection string from the *appsettings.json* file.
-
-<a name="cs"></a>
-
-## Add a database connection string
-
-Add a connection string to the *appsettings.json* file:
-
-# [Visual Studio](#tab/visual-studio)
-
-[!code-json[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/appsettings.json?highlight=10-12)]
-
-# [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
-
-[!code-json[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/appsettings_SQLite.json?highlight=10-12)]
-
----
-
 Build the project as a check for compiler errors.
 
 ## Scaffold movie pages
 
-Use the scaffolding tool to produce Create, Read, Update, and Delete (CRUD) pages for the movie model.
+Use the scaffolding tool to produce `Create`, `Read`, `Update`, and `Delete` (CRUD) pages for the movie model.
 
 # [Visual Studio](#tab/visual-studio)
 
 In **Solution Explorer**, right-click the *Controllers* folder **> Add > New Scaffolded Item**.
 
-![view of above step](adding-model/_static/add_controller21.png)
+![view of above step](adding-model/_static/add_controller5.png)
 
 In the **Add Scaffold** dialog, select **MVC Controller with views, using Entity Framework > Add**.
 
-![Add Scaffold dialog](adding-model/_static/add_scaffold21.png)
+![Add Scaffold dialog](adding-model/_static/add_scaffold5.png)
 
-Complete the **Add Controller** dialog:
+Complete the **Add MVC Controller with views, using Entity Framework** dialog:
 
-* **Model class:** *Movie (MvcMovie.Models)*
-* **Data context class:** *MvcMovieContext (MvcMovie.Data)*
+* In the **Model class** drop down, select **Movie (MvcMovie.Models)**.
+* In the **Data context class** row, select the **+** (plus) sign.
+  * In the **Add Data Context** dialog, the class name *MvcMovie.Data.MvcMovieContext* is generated, select **Add**.
 
 ![Add Data context](adding-model/_static/dc5.png)
 
-* **Views:** Keep the default of each option checked
-* **Controller name:** Keep the default *MoviesController*
-* Select **Add**
+* **Views** and **Controller name** : Keep the default.
+* Select **Add**.
 
 Visual Studio creates:
```

Comment:
Move this below all the tabs and replace it with

Scaffolding creates:

* bla
* bla

Scaffolding updates

* bla
* bla